lib/styles/components/_buttons.scss

```diff
@@ -0,0 +1,246 @@
+@import '../settings/animations';
+@import '../settings/colors';
+@import '../settings/spacings';
+@import '../settings/radiuses';
+@import '../settings/typography';
+@import '../settings/buttons';
+
+$default-lighter-shadow: 0 4px 12px rgba($color-firefly-black, 0.35);
+
+@mixin buttonColors(
+	$color: '',
+	$background: '',
+	$hover: '',
+	$focus: '',
+	$disabled: '',
+	$ripple: '',
+	$border: ''
+) {
+	@if $color != '' {
```

Comment:
should this color be optional? I have a feeling that when it's missing, then we have a bug